### DISCUSSION TITLE
Module '"vue-router/auto-routes"' has no exported member 'routes'

### DISCUSSION THREAD

**Discussion by NicolasMelin** [posva/unplugin-vue-router#505] [Questions and Help] [answered]
Hello all,

I have upgraded the module version 0.7.0 to 0.10.8, and now I have the following error :
```
Module '"vue-router/auto-routes"' has no exported member 'routes'
```

Before I had this import and it's works well :
```
import { routes } from 'vue-router/auto/routes';
```

and now I have this import :
```
import { routes } from 'vue-router/auto-routes';
```

The import `vue-router/auto-routes` have as target file the generated file `typed-router.d.ts`, I don't know if it's a good thing or not.

Thank's by advance for tou help and your support :).

Best regards

**Comment by posva** [comment, marked as answer]
You skipped a few versions, so I would recommend you to ensure the TS setup is done again: https://uvr.esm.is/introduction.html#setup. You can check the changelog for more info about the breaking changes, there were a few to fix TS compatibility issues

**Comment by NicolasMelin** [comment]
I have founded my issue :

I have missing this ts.config.ts option :
```{
  "compilerOptions": {
   ...
    "types": [
    ...
      "unplugin-vue-router/client"
    ...
    ]
}
```

Thank's for your help :).

**Reply by posva** [reply]
Using `types` has created problems in the past, I recommend you to use the `env.d.ts` version if possible (https://uvr.esm.is/introduction.html#setup)
